shiny-drum-machine/index.html

```diff
@@ -257,7 +257,7 @@
         </div>
     </section>
     
-    <section class='container active' id='tools'>
+    <section class='container' id='tools'>
         <div class="row buttons_container">
```

Comment:
Now I can see both (' and ") are being used in the code. We should be consistent. This could be done in follow-up PRs.